### PR TITLE
Fix oauth token deserialize issue

### PR DESCRIPTION
**Pull request by yisiqi**
OAuth tokens can't be deserialize, because all of these tokens' ExpirationPolicy has been wrapped by spring with CGILib.

I suggests we should avoid by defining spring bean to create any persisted entity.

### CODE REVIEW

**Comment by mmoayyed** [comment]
Looks good. thank you!

File: cas-server-support-oauth/src/main/java/org/apereo/cas/config/CasOAuthConfiguration.java [hunk 1]
```diff
@@ -254,15 +254,14 @@ public AccessTokenResponseGenerator oauthAccessTokenResponseGenerator() {
...
+    @RefreshScope
     public AccessTokenFactory defaultAccessTokenFactory() {
         final DefaultAccessTokenFactory f = new DefaultAccessTokenFactory();
         f.setAccessTokenIdGenerator(accessTokenIdGenerator());
         f.setExpirationPolicy(accessTokenExpirationPolicy());
         return f;
     }
 
-    @Bean
-    @RefreshScope
```
Comment (dima767) [inline cas-server-support-oauth/src/main/java/org/apereo/cas/config/CasOAuthConfiguration.java]: I'd then make this method `private`

File: cas-server-support-oauth/src/main/java/org/apereo/cas/config/CasOAuthConfiguration.java [hunk 2]
```diff
@@ -271,8 +270,6 @@ public ExpirationPolicy accessTokenExpirationPolicy() {
         );
     }
 
-    @Bean
-    @RefreshScope
     public ExpirationPolicy oAuthCodeExpirationPolicy() {
```
Comment (dima767) [inline cas-server-support-oauth/src/main/java/org/apereo/cas/config/CasOAuthConfiguration.java]: Make it `private`

**Comment by mmoayyed** [comment]
Thanks very much!

**Pull request merged**